l10n/en/products/vpn/shared.ftl

```diff
@@ -25,11 +25,20 @@ vpn-shared-features-servers = { $servers }+ servers in { $countries }+ countries
 
 vpn-shared-features-bandwidth = No bandwidth restrictions
 vpn-shared-features-activity = No logging of your network activity
+vpn-shared-features-activity-logs = No online activity logs now or ever
+
+# Variables:
+#   $countries (number) - number of available countries
+vpn-shared-features-access-countries = Access to servers in { $countries }+ countries
```

Comment:
Same as above, this is a very similar sentance to what we have translated already.

Reply:
Yeah. I think part of the challenge is trying to having different verbage for SEO purposes.